### ISSUE TITLE
Unable to Load Custom Properties in Word Online

### ISSUE THREAD

**Issue by andschleicher** [OfficeDev/office-js#328]
## Expected Behavior
Loading Custom Properties should function the same in the desktop version of Word and in Word Online.

## Current Behavior
Currently I am able to read in the custom properties for a document in the desktop version of Word without any issues, but if I attempt to load that same document in Word Online then the custom properties are read in as null. Previously this was working as expected but now I am unable to read in the properties again.

## Steps to Reproduce, or Live Example
This occurs for all Word Online documents with a custom property. I did find a workaround that forces the custom properties to be loaded and have detailed that below.

- Click on Edit in Word in the banner and open the file in Word desktop as prompted
- Make any change to the file and revert it, then save the document
- Close the desktop version of Word and click Resume Editing Here in Word Online

Upon performing these steps the custom properties load in Word Online as expected and the Add-In is able to function identically to the desktop version of Word.

## Context
We use custom properties to store information about the document that's then used to display information about the document in the Add-In, so the Add-In's functionality doesn't fully work in Word Online without the ability to read in custom properties.

## Your Environment

* Platform [PC desktop, Mac, iOS, Office Online]: Office Online
* Host [Excel, Word, PowerPoint, etc.]: Word
* Office version number: ______
* Operating System: Windows
* Browser (if using Office Online): Firefox

**Comment by JuaneloJuanelo** [comment]
@andschleicher I am not able to repro this issue. I need more details.
this is what i did 1 tried 2 scenarios:
1. creating a new document on Word for Windows.  On a newly created Word document i add 2 custom doc properties and save the document. Then i goto both One Drive for Consumer and/or One Drive for Business, upload the document. i open it from Word online: i am able to retrieve the custom doc properties.
2. creating the document in Word Online (on both consumer and business One Drive). crate document, then added 2 properties, close the document , open the document again in Online: properties are still there.

Again, I cannot repro your issue.  btw this is the code i am using to create/read them. please also share your code.
```

async function insertNumericProperty() {
    await Word.run(async (context) => {
        context.document.properties.customProperties.add("Numeric Property", 1234);

        await context.sync();
        console.log("Property added");
    });
}

async function insertStringProperty() {
    await Word.run(async (context) => {
        context.document.properties.customProperties.add("String Property", "Hello World!");

        await context.sync();
        console.log("Property added");
    });
}

async function readCustomDocumentProperties() {
    await Word.run(async (context) => {
        let properties = context.document.properties.customProperties;
        properties.load("key,type,value");

        await context.sync();
        for (var i = 0; i < properties.items.length; i++)
            console.log("Property Name:" + properties.items[i].key + "; Type=" + properties.items[i].type + "; Property Value=" + properties.items[i].value);
    });
}

```

**Comment by andschleicher** [comment]
@JuaneloJuanelo Thanks for the quick response and the code examples. After doing additional testing, the issue appears to be with how the custom properties are being added and not how they're being read. Our backend system adds identification properties to the documents as they're being downloaded and it appears that Word Online is having issues reading in those automatically added properties. However, as you mentioned it's able to read in any properties that I manually set on a file. If I do the workaround steps that I detailed in the first comment, it will be able to read in those automatically added properties in Word Online, but not until then.

Since the source of the issue appears to be dealing with our backend system rather than Office-js reading the custom properties, I'll close the ticket, but any suggestions as to why Office Online would be failing to read those properties when desktop Word can would be greatly appreciated. Thanks
